packages/entity-list/src/modules/preferences/sagas.js

```diff
@@ -34,9 +38,38 @@ export function* changePosition({payload}) {
 
   const newPositions = yield call(util.changePosition, positions, field, afterFieldPosition)
   yield put(setPositions(newPositions))
-  const listState = yield select(entityListSelector)
+  const listState = yield select(listSagas.entityListSelector)
   const formName = `${listState.formName}_list`
   yield call(rest.deleteUserPreferences, `${formName}.*.positions`)
   const positionPreferences = yield call(util.getPositionsPreferencesToSave, formName, newPositions)
   yield call(rest.savePreferences, positionPreferences)
 }
+
+export function* saveSorting() {
+  const {sorting: sortings} = yield select(listSagas.listSelector)
+  if (sortings.length > 0) {
+    const listState = yield select(listSagas.entityListSelector)
+    const formName = `${listState.formName}_list`
+    yield all([
```

Comment:
would  `yield call(rest.deleteUserPreferences, `${formName}.sorting*`)` work instead of two requests?

Reply:
I was afraid that a potential positioning of a field called "sorting_<something>" would also be deleted, and since this doesn't support full Regex, I can't specify that it should check if it is the end of the preference key.
Think this is relevant?